src/lib.rs

```diff
@@ -84,37 +84,38 @@ trait RenderPassInner<Ctx: Context> {
 }
 
 trait Context: Sized {
-    type AdapterId;
-    type DeviceId;
-    type QueueId;
-    type ShaderModuleId;
-    type BindGroupLayoutId;
-    type BindGroupId;
-    type TextureViewId;
-    type SamplerId;
-    type BufferId;
-    type TextureId;
-    type PipelineLayoutId;
-    type RenderPipelineId;
-    type ComputePipelineId;
-    type CommandEncoderId;
+    type AdapterId: Send + Sync;
+    type DeviceId: Send + Sync;
+    type QueueId: Send + Sync;
+    type ShaderModuleId: Send + Sync;
+    type BindGroupLayoutId: Send + Sync;
+    type BindGroupId: Send + Sync;
+    type TextureViewId: Send + Sync;
+    type SamplerId: Send + Sync;
+    type BufferId: Send + Sync;
+    type TextureId: Send + Sync;
+    type PipelineLayoutId: Send + Sync;
+    type RenderPipelineId: Send + Sync;
+    type ComputePipelineId: Send + Sync;
+    type CommandEncoderId: Send + Sync;
```

Comment:
we don't have send/sync on command encoders, so I don't think this line is needed